tests/src/system/basic/WskBasicTests.scala

```diff
@@ -348,14 +348,14 @@ class WskBasicTests
             assetHelper.withCleaner(wsk.rule, ruleName) {
                 (rule, name) =>
                     rule.create(name, trigger = triggerName, action = actionName)
-                    rule.create(name, trigger = triggerName, action = actionName, update = true)
+                    rule.create(name, trigger = triggerName, action = actionName, update = true, expectedExitCode = CONFLICT)
```

Comment:
But why is it a conflict in the first place? Updating has nothing to do with activating/deactivating (Update is PUT, activating/.. is POST).